identity/backend/src/test/resources/application-test.yaml

```diff
@@ -0,0 +1,2 @@
+spring:
+
```

Comment:
Do we need the file?

Reply:
I kept it as place holder so we have a file to put test related properties.